io_scene_nif/modules/nif_export/geometry/mesh/__init__.py

```diff
@@ -183,21 +172,23 @@ def export_tri_shapes(self, b_obj, n_parent, trishape_name=None):
                     mesh_hasspec = b_spec_prop
 
                 # gloss mat 'Hardness' scrollbar in Blender, takes values between 1 and 511 (MW -> 0.0 - 128.0)
-                mesh_mat_gloss = b_mat.specular_hardness
+                mesh_mat_gloss = b_mat.specular_intensity
 
                 # alpha mat
                 mesh_hasalpha = b_alpha_prop
-                mesh_mat_transparency = (1 - b_mat.alpha)
-                if b_mat.use_transparency:
+                mesh_mat_transparency = 1
+                if b_mat.blend_method != "OPAQUE":
                     if abs(mesh_mat_transparency - 1.0) > NifOp.props.epsilon:
                         mesh_hasalpha = True
-                elif mesh_hasvcola:
+                elif mesh_hasvcol:
                     mesh_hasalpha = True
                 elif b_mat.animation_data and 'Alpha' in b_mat.animation_data.action.fcurves:
                     mesh_hasalpha = True
 
                 # wire mat
-                mesh_haswire = (b_mat.type == 'WIRE')
+                # mesh_haswire = (b_mat.type == 'WIRE')
```

Comment:
I think they have some wireframe stuff, it might be a modifier.

Reply:
That's possible, I couldn't find a material setting for it.

Reply:
there is `b_obj.display_type = 'WIRE'` but I would prefer a dedicated setting.